commands/TenorCommand.ts

```diff
@@ -56,8 +58,8 @@ export class TenorCommand implements ISlashCommand {
                 },
                 author: {
                     icon: 'https://raw.githubusercontent.com/wreiske/Rocket.Chat.App-Tenor/master/images/Tenor-256.png',
-                    name: `/tenor ${trigger.trim()}`,
-                    link: `https://tenor.com/search/${trigger.trim()}`,
+                    name: (showLink ? `/tenor ${trigger.trim()} ${showLink}` : ''),
```

Comment:
I'm not exactly sure what you're trying to accomplish here... 

If showLink is true, it will show essentially '/tenor cats true', or nothing at all. What does this actually output?

Reply:
I put in some debugging outputs while trying to figure out the strange setting behavior described above. I just forgot to remove it.